src/protocols/aeternity/composables/aeTokenSwaps.ts

```diff
@@ -0,0 +1,93 @@
+import { computed, ref } from 'vue';
+import camelCaseKeysDeep from 'camelcase-keys-deep';
+
+import { ITokenSwap } from '@/types';
+import { NETWORK_TYPE_CUSTOM } from '@/constants';
+import { fetchAllPages, fetchJson, handleUnknownError } from '@/utils';
+import { useNetworks } from '@/composables';
+import { AE_TOKEN_SWAPS_URLS } from '@/protocols/aeternity/config';
+import { createPollingBasedOnMountedComponents } from '@/composables/composablesHelpers';
+
+let composableInitialized = false;
+
+const areTokenSwapsReady = ref(false);
+
+const tokenSwapsUrl = ref('');
+
+const POLLING_INTERVAL = 60000;
```

Comment:
I suggets to put the constants at the beginning and then states (avoid mixing them).